### ISSUE TITLE
[Bug]: Unable to function properly.

### ISSUE THREAD

**Issue by Pszz** [MetaMask/metamask-extension#24020]
### Describe the bug

When I click to switch network, I get this error.
I have tried reopening the browser and plug-in, but the problem still occurs. As a result, my wallet cannot be used.

<img width="368" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/12093551/319d6f07-d31c-4a8c-afc6-d8e93a8fa227">

<img width="804" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/12093551/d8aaca69-43d1-4ce2-9a06-b9c13a200120">

### Steps to reproduce

1. Click network list.
2. Error...

### Version

11.13.1

### Build type

None

### Browser

Chrome

### Operating system

MacOS

**Comment by gauthierpetetin** [comment]
Hi @Pszz , thanks for reporting this.
Does this happen on a specific network? (I see there's a question mark on your network)
Also which browser are you using?

**Comment by Pszz** [comment]
> Hi @Pszz , thanks for reporting this. Does this happen on a specific network? (I see there's a question mark on your network) Also which browser are you using?

Yes, a new network has been added. Since the RPC nodes in this network are unstable, you cannot switch to other normal networks when you are on this network.

**Comment by gauthierpetetin** [comment]
We're looking for a way to reproduce, is there any way you can tell us more about these RPC nodes?
Where did you get access to these?
Also which browser are you using?

**Comment by Pszz** [comment]
> We're looking for a way to reproduce, is there any way you can tell us more about these RPC nodes? Where did you get access to these? Also which browser are you using?

I'm using Chrome browser.

You can try to reproduce it
1. Add network
2. Enter name
3. When I entered the wrong RPC address, the confirm button got activated and when I pressed enter or clicked confirm, the wrong network was added.
4. This error occurs

**Comment by gauthierpetetin** [comment]
Hi @danjm , do you think this issue is related to this similar issue that was closed last year? https://github.com/MetaMask/metamask-extension/issues/20138

**Comment by darkwing** [comment]
Cannot reproduce this and since has not been seen.
